<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>music_widget</class>
 <widget class="QWidget" name="music_widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>941</width>
    <height>618</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">border-radius:8px;
background-color: rgb(255, 255, 255);</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>9</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QStackedWidget" name="stackedWidget">
       <property name="styleSheet">
        <string notr="true">boder:none;</string>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="page">
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <widget class="QLabel" name="label_songImage">
             <property name="minimumSize">
              <size>
               <width>130</width>
               <height>130</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>130</width>
               <height>130</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">
border-radius: 12px;</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="pixmap">
              <pixmap resource="myResources.qrc">:/player/images/player/pretty_crazy.jpg</pixmap>
             </property>
             <property name="scaledContents">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout">
             <item>
              <widget class="QLabel" name="label_Playlist">
               <property name="styleSheet">
                <string notr="true">color: rgb(0, 0, 0);
background: transparent;</string>
               </property>
               <property name="text">
                <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt; font-weight:700;&quot;&gt;士多啤梨啤梨冚家橙的歌单&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_Introduction">
               <property name="styleSheet">
                <string notr="true">color: rgb(0, 0, 0);
background: transparent;</string>
               </property>
               <property name="text">
                <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;简单介绍下你自己&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_5">
               <item>
                <widget class="QPushButton" name="pushButton_title_play">
                 <property name="minimumSize">
                  <size>
                   <width>90</width>
                   <height>30</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>90</width>
                   <height>30</height>
                  </size>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">QPushButton {
    background-color: #E3E3E3;
    color: black;
    border-radius: 12px;
    font-size: 15px;
    font-weight: bold;
    padding-left: 12px;
}
QPushButton:hover {
    background-color: #D4D4D4;
}
QPushButton:pressed {
    background-color: #D4D4D4;
}
</string>
                 </property>
                 <property name="text">
                  <string>播放</string>
                 </property>
                 <property name="icon">
                  <iconset resource="myResources.qrc">
                   <normaloff>:/player/images/player/play_button.png</normaloff>:/player/images/player/play_button.png</iconset>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="pushButton_add">
                 <property name="minimumSize">
                  <size>
                   <width>90</width>
                   <height>30</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>90</width>
                   <height>30</height>
                  </size>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">QPushButton {
    background-color: #E3E3E3;
    color: black;
    border-radius: 12px;
    font-size: 15px;
    font-weight: bold;
    padding-left: 12px;
}
QPushButton:hover {
    background-color: #D4D4D4;
}
QPushButton:pressed {
    background-color: #D4D4D4;
}
</string>
                 </property>
                 <property name="text">
                  <string>添加</string>
                 </property>
                 <property name="icon">
                  <iconset resource="myResources.qrc">
                   <normaloff>:/player/images/player/add_path.png</normaloff>:/player/images/player/add_path.png</iconset>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="pushButton_edit">
                 <property name="minimumSize">
                  <size>
                   <width>90</width>
                   <height>30</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>90</width>
                   <height>30</height>
                  </size>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">QPushButton {
    background-color: #E3E3E3;
    color: black;
    border-radius: 12px;
    font-size: 15px;
    font-weight: bold;
    padding-left: 12px;
}
QPushButton:hover {
    background-color: #D4D4D4;
}
QPushButton:pressed {
    background-color: #D4D4D4;
}
</string>
                 </property>
                 <property name="text">
                  <string>编辑</string>
                 </property>
                 <property name="icon">
                  <iconset resource="myResources.qrc">
                   <normaloff>:/player/images/player/edit_button1.png</normaloff>:/player/images/player/edit_button1.png</iconset>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="horizontalSpacer_9">
             <property name="orientation">
              <enum>Qt::Orientation::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item row="1" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_8">
           <item>
            <widget class="QPushButton" name="pushButton_title_songName">
             <property name="minimumSize">
              <size>
               <width>90</width>
               <height>30</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>90</width>
               <height>30</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton {
            border: none;
			color:black;
            background-color: transparent;
            font-size: 16px;
            padding-bottom: 4px; /* 为底线预留一点空间 */
        }
    QPushButton:hover {
	color:#00CC65;
        }


</string>
             </property>
             <property name="text">
              <string>歌曲</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_title_comment">
             <property name="minimumSize">
              <size>
               <width>90</width>
               <height>30</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>90</width>
               <height>30</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">
        QPushButton:hover {

color:#00CC65;
        }

QPushButton {
    border: none;
    background-color: transparent;
    font-size: 16px;
    color: #666;
    padding: 4px 12px 6px 12px; /* 左右 padding 增大，线会看起来短 */
}

QPushButton:checked {
    color: rgb(0,180,90);
    font-weight: bold;
    border-bottom: 2px solid rgb(0,180,90);
    padding: 4px 12px 4px 12px;
}</string>
             </property>
             <property name="text">
              <string>评论</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_7">
             <item>
              <widget class="QLineEdit" name="lineEdit_search">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>30</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>230</width>
                 <height>30</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">background-color: #E3E3E3;
border-raidus:3px;
</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_5">
               <property name="orientation">
                <enum>Qt::Orientation::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item row="2" column="0">
          <widget class="QStackedWidget" name="stackedWidget_song_comment">
           <widget class="QWidget" name="song">
            <layout class="QGridLayout" name="gridLayout">
             <item row="0" column="0">
              <layout class="QVBoxLayout" name="verticalLayout_4">
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_10">
                 <item>
                  <widget class="QLabel" name="label_Introduction_2">
                   <property name="styleSheet">
                    <string notr="true">color: #626262;
background: transparent;</string>
                   </property>
                   <property name="text">
                    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;歌名/歌手&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <spacer name="horizontalSpacer_2">
                   <property name="orientation">
                    <enum>Qt::Orientation::Horizontal</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>40</width>
                     <height>20</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                 <item>
                  <widget class="QLabel" name="label_Introduction_3">
                   <property name="styleSheet">
                    <string notr="true">color: #626262;
background: transparent;</string>
                   </property>
                   <property name="text">
                    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;专辑&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <spacer name="horizontalSpacer_3">
                   <property name="orientation">
                    <enum>Qt::Orientation::Horizontal</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>40</width>
                     <height>20</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                 <item>
                  <widget class="QLabel" name="label_Introduction_4">
                   <property name="styleSheet">
                    <string notr="true">color: #626262;
background: transparent;</string>
                   </property>
                   <property name="text">
                    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;时长&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <spacer name="horizontalSpacer_4">
                   <property name="orientation">
                    <enum>Qt::Orientation::Horizontal</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>40</width>
                     <height>20</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                </layout>
               </item>
               <item>
                <widget class="QListWidget" name="listWidget"/>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="comment"/>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="page_2"/>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="myResources.qrc"/>
 </resources>
 <connections/>
</ui>
